src/Init/Init_ByRestart_HDF5.cpp

```diff
@@ -625,16 +634,23 @@ void Init_ByRestart_HDF5( const char *FileName )
 // 3. load and allocate patches (load particles as well if PARTICLE is on)
    if ( MPI_Rank == 0 )    Aux_Message( stdout, "   Loading patches and particles ...\n" );
 
+   int NCompStore = NCOMP_TOTAL;
+
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
```

Comment:
Should we also load `STUB` field when `GAMER_DEBUG` is both enabled in the restart file and the current GAMER?

Reply:
No. This field is only used in post-processing for debugging purposes.